infrastructure/lambda/template.yaml

```diff
@@ -669,10 +667,10 @@ Resources:
         - Statement:
             - Effect: Allow
               Action:
-                - ssm:GetParameter
+                - ssm:GetParametersByPath
               Resource:
                 - !Sub
-                  - "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${PREFIX}/HMPODVAD*"
+                  - "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${PREFIX}/HMPODVAD/HttpClient"
```

Comment:
Have we confirmed/tested that this doesn't cause any permissions issues?

Reply:
This reduces the parameters that CertExpiryReminder has access to, to only the path with the certs.